src/test/kotlin/se/uulm/snowballr/backend/service/criterion/CreateCriterionTest.kt

```diff
@@ -12,17 +12,39 @@ import org.junit.jupiter.api.assertThrows
 import se.uulm.snowballr.backend.DataBuilder
 import se.uulm.snowballr.backend.TestSpecificException
 import se.uulm.snowballr.backend.auth.GrpcContext
+import se.uulm.snowballr.backend.model.SnowballRException
 import se.uulm.snowballr.backend.service.MainServiceTest
 import snowballr.CriterionOuterClass
+import snowballr.UserOuterClass.UserRole
 import java.util.UUID
 
 @ExperimentalCoroutinesApi
 @DelicateCoroutinesApi
 class CreateCriterionTest : MainServiceTest() {
+    private val dummyUserUUID = UUID.randomUUID()
+
     @BeforeEach
-    fun setUpTest() {
+    fun setupTest() {
         every { GrpcContext.getUserIdFromContext() } throws NotImplementedError()
+        coEvery { userRepoMock.getUserById(GrpcContext.getUserIdFromContext()) } throws NotImplementedError()
```

Comment:
The other call is mocked one call above.
```suggestion
        coEvery { userRepoMock.getUserById(any()) } throws NotImplementedError()
```